addons/website_sale/models/website.py

```diff
@@ -418,7 +418,7 @@ def _get_current_pricelist_id(self, partner_sudo):
 
     @api.model
     def _get_current_fiscal_position_id(self, partner_sudo):
-        AccountFiscalPosition = self.env['account.fiscal.position']
+        AccountFiscalPosition = self.env['account.fiscal.position'].sudo()
```

Comment:
This file diff is unrelated (and already fixed in master anyway) :eyes:

Reply:
I told you that I changed the files :p Now I see why I didn't find it after...